test/e2e/framework/pod/resize.go

```diff
@@ -456,6 +456,14 @@ func WaitForPodResizeActuation(ctx context.Context, f *framework.Framework, podC
 func ExpectPodResized(ctx context.Context, f *framework.Framework, resizedPod *v1.Pod, expectedContainers []ResizableContainerInfo) {
 	ginkgo.GinkgoHelper()
 
+	// Cgroup Values are verified by execing into the containers, so we first wait for them to have started.
+	for i := range resizedPod.Status.ContainerStatuses {
+		WaitForPodContainerStarted(ctx, f.ClientSet, resizedPod.Namespace, resizedPod.Name, i, f.Timeouts.PodStart)
```

Comment:
`i` is the index of the container status here, not the container spec. Unfortunately the sort order of those doesn't match.